lib/rubocop/cop/mixin/check_line_breakable.rb

```diff
@@ -216,6 +218,8 @@ def process_args(args)
 
       # @api private
       def already_on_multiple_lines?(node)
+        return node.first_line != node.arguments.last.last_line if node.def_type?
```

Comment:
This is needed as `node.last_line` returns the line where the `end` keyword is. Maybe there is a better approach here, the downside of the approach I chose is that:

```ruby
def foo(bar, bat
)

end
```

Does get wrapped on multiple lines. That's not strictly a bad behaviour as this only triggers when the line is longer then the maximum length, but still somewhat unexpected.